src/celengine/render.h

```diff
@@ -208,16 +208,15 @@ class Renderer
         ShowEclipticGrid        = 0x0000000000800000,
         ShowHorizonGrid         = 0x0000000001000000,
         ShowEcliptic            = 0x0000000002000000,
-        ShowTintedIllumination  = 0x0000000004000000,
```

Comment:
maybe better to leave a hole and use it later for any new flag?

Reply:
Sounds reasonable, will update this.